ipywidgets/widgets/widget.py

```diff
@@ -223,6 +272,10 @@ def open(self):
             if buffers:
                 # FIXME: workaround ipykernel missing binary message support in open-on-init
                 # send state with binary elements as second message
+                # TODO: if this gets fixed, _split_state_buffers does not need to have a seperate
```

Comment:
Cool, the GitHub UI shows which tags the commit is part of - shows that the fix is in ipykernel 4.2 and later...